src/main/java/org/candlepin/model/Pool.java

```diff
@@ -1309,15 +1303,13 @@ public String toString() {
     protected void onCreate() {
         super.onCreate();
 
-        this.synchronizeProductUuid();
         this.type = this.getType();
     }
 
     @Override
     protected void onUpdate() {
         super.onCreate();
```

Comment:
Not your bug, but we should fix this while we're here: that should be calling `super.onUpdate()`, not onCreate as it's doing now.

Reply:
Oh haha I looked at the multiple times and didn't realize that was an `onCreate` method being used in the `onUpdate` method. Updated